src/app/course/_services/course-event.service.ts

```diff
@@ -34,7 +34,7 @@ export class CourseEventService {
             .delete<CourseEvent>(url)
             .pipe(catchError(
                 this.apiService.handleError<CourseEvent>(
-                    `Error occurred while deleting Course Event`
+                    `Error occurred while deleting Course Assessment`
```

Comment:
I believe our convention for the error message is to write it as a sentence, therefore only the first letter should be capitalized.
```suggestion
                    `Error occurred while deleting course assessment`
```